### ISSUE TITLE
might consider using something more efficient than CSV

### ISSUE THREAD

**Issue by raspyweather** [hadikcz/internet-speed-continuous-monitor#1]
Probably it's a good idea to use something more space efficient than CSV, as it will take lots of storage and lead to problems having to handle huge files after a while.

**Comment by ItsEmilyJpg** [comment]
Yeah, I will add JSON support, but keep CSV because its easy to import to the Excel or other sheets softer.

Thanks for issue :)
